src/System.Diagnostics.Process/src/System/Diagnostics/ProcessManager.Linux.cs

```diff
@@ -115,16 +115,15 @@ internal static ProcessInfo CreateProcessInfo(int pid, ReusableTextReader reusab
                 reusableReader = new ReusableTextReader();
             }
 
-            Interop.procfs.ParsedStat stat;
-            return Interop.procfs.TryReadStatFile(pid, out stat, reusableReader) ?
-                CreateProcessInfo(ref stat, reusableReader) :
-                null;
+            return Interop.procfs.TryReadStatFile(pid, out Interop.procfs.ParsedStat stat, reusableReader) &&
+                   Interop.procfs.TryReadStatusFile(pid, out Interop.procfs.ParsedStatus status, reusableReader) ?
```

Comment:
It may not matter in practice, but it seems like we shouldn't need to fail the call if we can't successfully open/parse the second file.

Reply:
Done.